course02/homework01/linked_list.h

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+
+using namespace std;
+
+class LinkedList
+{
+
+public:
+	LinkedList();
+	~LinkedList();
```

Comment:
You forgot the rule of three and just got a nasty bug because you did use pointers even though I told you several times not to do so! You should have defined a copy constructor and assignment operator. If I copy your list, your problem will UB.